boards/shared/third_party/TraceRecorder/streamports/Jlink_RTT/include/trcStreamingPort.h

```diff
@@ -71,7 +71,7 @@ extern "C" {
  * But integrated J-Link OB interfaces are slower and needs about 5-10 KB, 
  * depending on the amount of data produced.
  ******************************************************************************/
-#define TRC_CFG_RTT_BUFFER_SIZE_UP 2000
+#define TRC_CFG_RTT_BUFFER_SIZE_UP 5000
```

Comment:
Now that we have more RAM on the boards we can afford larger buffer for Tracealyzer

Reply:
Fine by me, we should just keep this in mind if we ever need to free up RAM, given how much tracelyzer takes up.